clusters/build-clusters/01_cluster/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -82,3 +82,31 @@ items:
     - get
     - list
     - watch
+
+- kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: ci-admins-secret
+  rules:
+    - apiGroups:
+        - ""
+      resources:
+        - secrets
+      verbs:
+        - get
+        - create
+        - update
+        - patch
+
+- kind: ClusterRoleBinding
```

Comment:
Lets please stop creating anymore CRBs and instead have exactly one and use the aggregation feature: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles

Reply:
Updated.